events/events.go

```diff
@@ -137,6 +137,7 @@ type XRPCStreamEvent struct {
 	Error         *ErrorFrame
 	RepoCommit    *comatproto.SyncSubscribeRepos_Commit
 	RepoHandle    *comatproto.SyncSubscribeRepos_Handle
+	Identity      *comatproto.SyncSubscribeRepos_Identity
```

Comment:
It's true that the semantics of "RepoIdentity" would be a bit weird b/c identity kind of transcends a specific repo or account (?), but feels like we should stay consistent here and keep the `Repo` prefix?

Reply:
I guess we can, it just seemed a bit wonky cause it's related to the PLC identity not the repo necessarily.

I'm fin with it either way